app/root.tsx

```diff
@@ -0,0 +1,140 @@
+import { Link, Links, Meta, NavLink, Outlet, Scripts, ScrollRestoration } from '@remix-run/react';
+import type { LinksFunction } from '@remix-run/node';
+
+import './tailwind.css';
+import { Logo } from './components/Logo';
+import { ArrowUpRight, History, Search } from 'lucide-react';
+import { useEffect } from 'react';
+
+export const links: LinksFunction = () => [];
+
+const nav = [
+  {
+    title: 'All Releases',
+    path: '/release',
+  },
+  {
+    title: (
+      <>
+        <History className="w-4 h-4" />
+        History
+      </>
+    ),
+    path: '/history',
+  },
+  {
+    title: (
+      <>
+        <Search className="w-4 h-4" />
+        PR Lookup
+      </>
+    ),
+    path: '/pr-lookup',
+  },
+  {
+    title: (
+      <>
+        Docs
+        <ArrowUpRight className="w-3 h-3" />
+      </>
+    ),
+    path: 'https://electronjs.org/docs',
```

Comment:
```suggestion
    path: 'https://www.electronjs.org/docs/latest',
```

nit: nice to avoid the redirect